numba/npyufunc/wrappers.py

```diff
@@ -1,13 +1,19 @@
 from __future__ import print_function, division, absolute_import
+
+from collections import namedtuple
+
 import numpy as np
 
 from llvmlite.llvmpy.core import Type, Builder, ICMP_EQ, Constant
 
-from numba import types, cgutils, compiler
+from numba import types, cgutils
 from numba.compiler_lock import global_compiler_lock
 from ..caching import make_library_cache, NullCache
 
 
+_wrapper_info = namedtuple('_wrapper_info', ['library', 'env', 'name'])
```

Comment:
This is multiply defined (other definition https://github.com/numba/numba/pull/4155/files#diff-ec5b6bd5010b2b32f80834aad542033cR32).  The wrapper info seems sufficiently consistent so it seems like it ought to go somewhere shared?

Reply:
dropped the one in parallel.py and make it reference the one in this file.